### ISSUE TITLE
Warning about QTWEBENGINE_CHROMIUM_FLAGS tells user to use qt.args for chromium options, but does not say how, and there is no documentation

### ISSUE THREAD

**Issue by a-m-joseph** [qutebrowser/qutebrowser#6652]
You have QTWEBENGINE_CHROMIUM_FLAGS=' --js-flags=--noexpose_wasm ...' set in your environment. This is currently unsupported and interferes with qutebrowser's own flag handling (including workarounds for certain crashes). Consider using the qt.args qutebrowser setting instead.

Qutebrowser's documentation does not explain how to set **_chromium_** options via qt.args.  It only explains how to set "qt flags" that way, but does not explain how to ask qt to ask chromium to ask lasagna to ask chromium to set the flag.

Also, after an hour of fighting with this thing, I still cannot figure out how to disable WASM using --qt-flag or --qt-arg.  So I gave up and kept using QTWEBENGINE_CHROMIUM_FLAGS.

When displaying this error message you should parse the user's $QTWEBENGINE_CHROMIUM_FLAGS and print out the recommended --qt-foo command line arguments that you want the user to add in place of the environment variable.

**Comment by The-Compiler** [comment]
I don't follow. `:help qt.args` says:

> Additional arguments to pass to Qt, without leading --. With QtWebEngine, some Chromium arguments (see https://peter.sh/experiments/chromium-command-line-switches/ for a list) will work.

Thus, did you try `js-flags=--noexpose_wasm ...` in `qt.args`? Note that the `--qt-flag` and `--qt-arg` arguments are something different.

**Comment by a-m-joseph** [comment]
Oh my, this is very confusing.  The "--qt-arg" command line option doesn't do the same thing as the "qt.args" configuration option?  Did I read that correctly?

Is there some way this can be done with a command line option or environment variable?

I use qutebrowser on several machines which all have different graphics hardware, each necessitating different QTWEBENGINE_CHROMIUM_FLAGS.  They all use the same config.py, which is under version control.  So there isn't a single setting that I can hardwire into the configuration file.  It's sounding like "qt.args" isn't really a replacement for QTWEBENGINE_CHROMIUM_FLAGS after all.

**Comment by The-Compiler** [comment]
> Oh my, this is very confusing. The "--qt-arg" command line option doesn't do the same thing as the "qt.args" configuration option? Did I read that correctly?

Right. In fact, the `qt.args` setting is the equivalent to `--qt-flag`. See #6065.

> Is there some way this can be done with a command line option or environment variable?

Yes, via `--qt-flag "js-flags=--noexpose_wasm ..."`. You can also set any setting via the command line using `-s`.

> I use qutebrowser on several machines which all have different graphics hardware, each necessitating different QTWEBENGINE_CHROMIUM_FLAGS. They all use the same config.py, which is under version control. So there isn't a single setting that I can hardwire into the configuration file.

Given that `config.py` is a Python file, you can base your settings on whatever conditionals you see fit (e.g. a hostname).
